problems/getCountriesSortedByPopulation.js

```diff
@@ -10,8 +10,20 @@
 * 
 */
 
-function getCountriesSortedByPopulation() {
-
+function getCountriesSortedByPopulation(arr) {
+    let newArr = []
+    let finalArr = []
+    let i = 0
+    let a = 0
+    while(i < arr.length){
+        newArr[i] = arr[i]
+        i++
+    }
+    newArr.sort((a, b) => b.population - a.population)
+    for(let i in newArr) {
+        finalArr[i] = newArr[i].country
+    }
```

Comment:
Which array method can you use to return a new array with updated information inside? This array method will replace the two loops you have here.